.github/workflows/pr.yaml

```diff
@@ -102,7 +102,7 @@ jobs:
         fi
         cd test
         radare2 -N -Qc 'e cfg.newshell' -
-        make
+        make && make fuzz-tests
```

Comment:
@XVilka @trufae maybe we don't want to run fuzz-tests on each PR ? it does ~11 minutes to -test builds.